tutoraspects/plugin.py

```diff
@@ -276,7 +245,6 @@
         ("SUPERSET_LMS_EMAIL", "superset/lms-admin@aspects.invalid"),
         ("SUPERSET_OWNERS", []),
         # Set to 0 to have no row limit.
-        ("SUPERSET_ROW_LIMIT", 100_000),
```

Comment:
I think we should keep this one too, unless there's an easy patch for it. I suspect it's going to be one of the things people want to override for performance.

Reply:
the easy fix to use the patch `superset-config`